src/dataService/savedItemsService.ts

```diff
@@ -393,6 +393,20 @@ export class SavedItemDataService {
     return this.listItemUpdateBuilder().where('item_id', itemId);
   }
 
+  /**
+   * Get saved item IDs for a given user.
+   * @param offset
+   * @param limit
+   */
+  public getSavedItemIds(offset: number, limit: number) {
+    return this.db('list')
```

Comment:
related/unrelated -- I like the idea of having a factory pattern to make all our queries safer, which always includes `andWhere('user_id', this.userId)`

Reply:
don't change anything rn, just wanted to write down a thought lol